### ISSUE TITLE
Preference option to display emoji shortcodes instead of emoji

### ISSUE THREAD

**Issue by lawremipsum** [florence-social/mastodon-fork#75]
Users on MSP Social frequently complain that emoji are too small to read—especially some custom emoji—and this is even after I increased their size by about 50%.

Users should be able to set a preference that will let them just see all emojis as shortcodes instead. This would be an accessibility improvement for everyone that has trouble making out those tiny pictograms.

**Comment by Feufochmar** [comment]
To do this, you also need to decode the standard emoji into shortcodes. This can be hard to do, as some non-pictogram characters can also be emoji, like ™ vs :tm: (though this one is a bad example on Mastodon).

When you input standard emojis, the text stored in databases and federated over ActivityPub contains characters of the pictogram unicode blocks, not :emoji_shortcode: (:emoji_shortcode: is only used for custom emojis). Currently, a client that do not explicitely support emoji will still display standard emoji as pictograms if it has fonts supporting emoji installed (potentially in black and white), while custom emoji will be displayed as :emoji_shortcode:.

**Comment by ghost** [comment]
mastodon does already implement a library of emoji and shortcodes, though, so replacing it with text when you *do* correctly detect that a character is an emoji (which is a solveable problem) is definitely possible.

the option only being for custom emoji could also be totally good, tbh

**Comment by ghost** [comment]
er, we mean: it would good to have a separate option that says "only turn custom emoji into shortcodes" attached to this option
